tools/build_pytorch_libs.sh

```diff
@@ -261,6 +261,7 @@ function build_caffe2() {
       -DBUILD_PYTHON=$BUILD_PYTHON \
       -DBUILD_SHARED_LIBS=$BUILD_SHARED_LIBS \
       -DBUILD_BINARY=$BUILD_BINARY \
+      -DBUILD_TORCH_TEST=$BUILD_TEST \
```

Comment:
For simplifying macros, could you change BUILD_TORCH_TEST to BUILD_TEST directly? I think only place it is invoked is here: https://github.com/pytorch/pytorch/blob/4bf5fc44c86d835f4812e7b3baffae9c65de0677/torch/CMakeLists.txt

Reply:
But see https://github.com/pytorch/pytorch/pull/11431 for where we might want the opposite flag name (more specifci).

Reply:
Both are good points. Eventually we might want to switch all over to PT_ prefix (or similar), but for now I'll switch it to BUILD_TEST so that we are consist.